untwine/Common.cpp

```diff
@@ -129,7 +133,11 @@ std::vector<std::string> directoryList(const std::string& dir)
         fs::directory_iterator end;
         while (it != end)
         {
+#ifndef __MINGW32__
```

Comment:
I think that `string()` is totally sufficient. The macro tests can go away, along with the utwine::fromNative() call. But this needs to be testsed.

Reply:
string() doesn't work with MSVC compiler (It asks for wstring())

Reply:
I will check.